src/Tooltip/tooltip.scss

```diff
@@ -47,91 +42,109 @@ $arrow-color: $brc-darkGrey !default;
     font-weight: 400;
   }
 
-  &:after, &:before {
-    border: solid transparent;
+  &::after,
+  &::before {
```

Comment:
It's not to support old browsers, is to distinguish the [`pseudo-elements`](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements) from the [`pseudo-classes`](https://developer.mozilla.org/en-US/docs/Web/CSS/pseudo-classes).